apps/src/lab2/views/components/resizeBar.module.scss

```diff
@@ -0,0 +1,55 @@
+@import 'color.scss';
+
+$bar-width: 1px;
+$bar-width-grab-area: 5px;
+
+$grab-area-offset: calc(($bar-width-grab-area - 1px)/2);
```

Comment:
Nit: could refer to `$bar-width` in calculation 
`$grab-area-offset: calc(($bar-width-grab-area - $bar-width)/2);`

Reply:
Thanks for updating to use variable name after discussion offline.